### ISSUE TITLE
Sidebar always switched ON

### ISSUE THREAD

**Issue by KlausBlum** [xournalpp/xournalpp#5127]
### Operating System

Windows

### (Linux Only) Display Server

None

### Installation Method

Windows installer from official release

### Xournal++ Version

1.2.1 and 1.2.0

### libgtk Version

3.24.34

### Bug Description

When leaving full-page mode and going back to normal viewing mode, the sidebar is always turned on. 
There is no way to prevent this. I've switched back to 1.1.3 to get rid of this behavior. 

### Expected Behaviour

Last setting for sidebar (on/off) should be remembered.

### Steps to Reproduce

1. Switch off sidebar (if visible). [F12]
2. Turn on fullscreen mode. [F11]
3. Turn off fullscreen mode. [F11] 
Now sidebar is visible again.

**Comment by rolandlo** [comment]
I can confirm the issue. This code looks wrong:
https://github.com/xournalpp/xournalpp/blob/19618638b4f50d9ea92a9896059199da1ca7945a/src/core/control/FullscreenHandler.cpp#L30-L34
It should be easy to fix though.

**Comment by tmoerschell** [comment]
It looks like most of this file is old code that isn't used anymore. Sidebar (along with menubar and toolbar) visibility is handled in Control::loadViewMode() (which is called when entering or exiting fullscreen or presentation mode), and which in turn shows or hides the widgets based on the settings. Those settings are reset using Settings::loadViewMode(). 
So we need to change the default view mode when widget visibility changes in default mode. 
https://github.com/xournalpp/xournalpp/blob/19618638b4f50d9ea92a9896059199da1ca7945a/src/core/gui/MainWindow.cpp#L451C1-L451C1
https://github.com/xournalpp/xournalpp/blob/19618638b4f50d9ea92a9896059199da1ca7945a/src/core/gui/MainWindow.cpp#L371C1-L385C2
Should we change settings as well? It isn't done right now

**Comment by tmoerschell** [comment]
Found out something more: there seems to be currently no way to edit the defaultViewModeAttributes setting (in the gui), so it is always loaded the same at startup. 

@KlausBlum this allows for a quick fix for you though, so you don't have to wait for a pull request: 
Open C:\Users\<user>\AppData\Local\xournalpp\settings.xml, find the line with `"defaultViewModeAttributes"`, and remove `showSidebar` from the line. It should look like this:
```html
    <!--Which GUI elements are shown in default view mode, separated by a colon (,)-->
    <property name="defaultViewModeAttributes" value="showMenubar,showToolbar"/>
```
And that should fix it for you

**Comment by KlausBlum** [comment]
@tmoerschell Yes, that works. Thanks for your help! :relaxed:

**Comment by tmoerschell** [comment]
This issue can be closed now as it was solved by #5130
